### ISSUE TITLE
Challenges working with MATLAB tutorial

### ISSUE THREAD

**Issue by KirstieJane** [RainCloudPlots/RainCloudPlots#9]
Copied from feedback at PeerJ: https://peerj.com/preprints/27137v1/#feedback-1186

> Great resource, thank you. Just working through the MATLAB tutorial raincloudplotstutorial.mlx. The first section that generates codedir, figdir, and datadir has a couple of bugs - it uses double quotes instead of single quotes around the directory names, and a '+' in the square brackets which should be dropped. Cheers, Alex

This may be covered in the discussions in #7. Happy to take any advice on the best way to structure these conversations 😸

**Comment by TimVanMourik** [comment]
Exactly, as mentioned this is a backwards compatibility issue and not so much a bug. I'm making a legacy MATLAB script in #10 that should go back to at least MATLAB 2015a.

**Comment by KirstieJane** [comment]
:+1: Gotcha. Happy to remove the bug label if anyone feels strongly about it. Making it backwards compatible is great though! Thanks @TimVanMourik ✨

**Comment by TimVanMourik** [comment]
I don't feel too strong about labels in general 😎
